content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Control Topology Management Policies on a node
+reviewers:
+- lmdaly
+- nolancon
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+
+An increasing number of systems leverage a combination of CPUs and hardware accelerators to support latency-critical execution and high-throughput parallel computation. These include workloads in fields such as telecommunications, scientific computing, machine learning, financial services and data analytics. Such hybrid systems comprise a high performance environment.
+
+In order to extract the best performance, optimizations related to CPU isolation and memory and device locality are required. However, in Kubernetes, these optimizations are handled by a disjoint set of components.
+
+Topology Manager is a component in Kubelet that provides node level policies to enable these performance optimizations in an user abstract manner.
+
+{{% /capture %}}
+
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Topology Manager
+
+Currently, the CPU and Device Manager in Kubernetes make resource allocation decisions independently of each other.
+This can result in a sub-optimal allocations on multiple socketed systems, performance/latency sensitive applications
+will suffer due to these sub-optimal allocations.
+
+The Topology Manager introduces a algorithm to allow components that care about Topology Aware resource allocation to provide
+a list of their resources with the associated socket. This allows the Topology Manager to calculate the best possible socket for
+the incoming container and store this. The resource allocators can then consult this when making their resource allocation decisions.
+
+### Configuration
+
+The Topology Manager is an alpha feature in Kubernetes v1.15. 
+
+The Topology Manager currently works only on a node with the `static` CPU Manager
+Policy enabled and on Pods in the Guaranteed QOS. Topology Manager will align CPU and device requests.
+
+Topology Manager has two allocation policies, that are set via a Kubelet flag --topology-manager-policy.
```

Comment:
you may want to quote "--topology-manager-policy" using a pair of backticks to make sure it will be rendered properly.